modules/distribution/src/repository/resources/dbscripts/apimgt/h2.sql

```diff
@@ -263,112 +265,111 @@ CREATE TABLE IF NOT EXISTS SP_FEDERATED_IDP (
+ 			IS_LOCAL_CLAIM_DIALECT CHAR(1) NOT NULL DEFAULT '0',
+ 			DISPLAY_NAME VARCHAR(255),
+			PRIMARY KEY (ID),
+			UNIQUE (TENANT_ID, NAME));
 INSERT INTO IDP (TENANT_ID, NAME, HOME_REALM_ID) VALUES (-1234, 'LOCAL', 'localhost');
 
 CREATE TABLE IF NOT EXISTS IDP_ROLE (
-                  ID INTEGER AUTO_INCREMENT,
-                  IDP_ID INTEGER,
-                  TENANT_ID INTEGER,
-                  ROLE VARCHAR(254),
-                  PRIMARY KEY (ID),
-                  UNIQUE (IDP_ID, ROLE),
-                  FOREIGN KEY (IDP_ID) REFERENCES IDP(ID) ON DELETE CASCADE);
+			ID INTEGER AUTO_INCREMENT,
+			IDP_ID INTEGER,
+			TENANT_ID INTEGER,
+			ROLE VARCHAR(254),
+			PRIMARY KEY (ID),
+			UNIQUE (IDP_ID, ROLE),
+			FOREIGN KEY (IDP_ID) REFERENCES IDP(ID) ON DELETE CASCADE);
 
 CREATE TABLE IF NOT EXISTS IDP_ROLE_MAPPING (
-                  ID INTEGER AUTO_INCREMENT,
-                  IDP_ROLE_ID INTEGER,
-                  TENANT_ID INTEGER,
-                  USER_STORE_ID VARCHAR (253),
-                  LOCAL_ROLE VARCHAR(253),
-                  PRIMARY KEY (ID),
-                  UNIQUE (IDP_ROLE_ID, TENANT_ID, USER_STORE_ID, LOCAL_ROLE),
-                  FOREIGN KEY (IDP_ROLE_ID) REFERENCES IDP_ROLE(ID) ON DELETE CASCADE);
+			ID INTEGER AUTO_INCREMENT,
+			IDP_ROLE_ID INTEGER,
+			TENANT_ID INTEGER,
+			USER_STORE_ID VARCHAR (253),
+			LOCAL_ROLE VARCHAR(253),
+			PRIMARY KEY (ID),
+			UNIQUE (IDP_ROLE_ID, TENANT_ID, USER_STORE_ID, LOCAL_ROLE),
+			FOREIGN KEY (IDP_ROLE_ID) REFERENCES IDP_ROLE(ID) ON DELETE CASCADE);
 
 CREATE TABLE IF NOT EXISTS IDP_CLAIM (
-                  ID INTEGER AUTO_INCREMENT,
-                  IDP_ID INTEGER,
-                  TENANT_ID INTEGER,
-                  CLAIM VARCHAR(254),
-                  PRIMARY KEY (ID),
-                  UNIQUE (IDP_ID, CLAIM),
-                  FOREIGN KEY (IDP_ID) REFERENCES IDP(ID) ON DELETE CASCADE);
+			ID INTEGER AUTO_INCREMENT,
+			IDP_ID INTEGER,
+			TENANT_ID INTEGER,
+			CLAIM VARCHAR(254),
+			PRIMARY KEY (ID),
+			UNIQUE (IDP_ID, CLAIM),
+			FOREIGN KEY (IDP_ID) REFERENCES IDP(ID) ON DELETE CASCADE);
 
 CREATE TABLE IF NOT EXISTS IDP_CLAIM_MAPPING (
-                  ID INTEGER AUTO_INCREMENT,
-                  IDP_CLAIM_ID INTEGER,
-                  TENANT_ID INTEGER,
-                  LOCAL_CLAIM VARCHAR(253),
-                  DEFAULT_VALUE VARCHAR(255),
-                  IS_REQUESTED VARCHAR(128) DEFAULT '0',
-                  PRIMARY KEY (ID),
-                  UNIQUE (IDP_CLAIM_ID, TENANT_ID, LOCAL_CLAIM),
-                  FOREIGN KEY (IDP_CLAIM_ID) REFERENCES IDP_CLAIM(ID) ON DELETE CASCADE);
+			ID INTEGER AUTO_INCREMENT,
+			IDP_CLAIM_ID INTEGER,
+			TENANT_ID INTEGER,
+			LOCAL_CLAIM VARCHAR(253),
+			DEFAULT_VALUE VARCHAR(255),
+			IS_REQUESTED VARCHAR(128) DEFAULT '0',
+			PRIMARY KEY (ID),
+			UNIQUE (IDP_CLAIM_ID, TENANT_ID, LOCAL_CLAIM),
+			FOREIGN KEY (IDP_CLAIM_ID) REFERENCES IDP_CLAIM(ID) ON DELETE CASCADE);
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04